catalog/ui/catalog-ui-search/src/main/webapp/component/visualization-selector/visualization-selector.less

```diff
@@ -1,31 +1,28 @@
 @{customElementNamespace}visualization-selector {
 
-    > div {
-        padding: @minimumSpacing;
-    }
+  .visualization-choice {
+    white-space: nowrap;
+    padding: @largeSpacing;
+    cursor: move; /* fallback if grab cursor is unsupported */
```

Comment:
This cursor fallback is a perfect opportunity for a less mixin.

In addition, there are likely other preexisting areas of the code that would benefit from a cursor mixin existing.

Reply:
Done.